blog/relacja-z-meetupu-hitachi-karate/index.md

```diff
@@ -0,0 +1,84 @@
+
+![](images/patrycja_jap.jpg)
+
+[Patrycja Pyrek](https://www.linkedin.com/in/patrycja-pyrek/), jako znawczyni tematu, wprowadziła nas we frapujące meandry kultury japońskiej, którą poznała z pierwszej ręki, kilkakrotnie odwiedzając Kraj Kwitnącej Wiśni i zgłębiając miejscowe zwyczaje. Słuchacze dowiedzieli się między innymi o tym, dlaczego nie należy wbijać pałeczek w ryż podczas posiłku oraz jak rozumieć szyld z godzinami otwarcia japońskiego sklepu od "23:00 - 25:00" :-)
+
+Arigatō, Patrycja!
+
+
+### Tech writer mistrzem wywierania wpływu
+
+![](images/1694696569715.jpg)
+
+W trakcie Meetupu [Monika Dąbrowska](https://www.linkedin.com/in/monika-klimek1/) opowiedziała o tym, co profesor Cialdini - specjalista na skalę światową w dziedzinie wywierania wpływu - może doradzić pisarzom technicznym, którzy mają problem z uzyskaniem uwag ekspertów domenowych co do opracowanego dokumentu. W codziennych bojach, często zwracamy uwagę na narzędzia, problemy techniczne i organizację pracy, a nie należy zapominać również o budowaniu zdrowych relacji ze współpracownikami. Monika przedstawiła nam perspektywę na mechanizmy rządzące budowaniem relacji między ludźmi. Otrzymaliśmy kilka praktycznych wskazówek, które mogą pomóc podczas codziennej pracy dokumentalisty. 
+Warto tu zaznaczyć, że choć tematyka prac Cialdiniego niektórym może przywodzić na myśl techniki makiawelicznej manipulacji, to znajomość uniwersalnych zasad, którymi rządzi się ludzka natura, pomaga nam lepiej rozumieć samych siebie i tę drugą osobę odgrywającą rolę *Subject Matter Expert*. Można spokojnie korzystać z tej wiedzy, mając krystalicznie czyste intencje i bez robienia kogokolwiek w bambuko :) 
+
+### Wszystko ma swoje priorytety
+
+![](images/andy_prez.jpg)
+
+Mimo opornego projektora, który przestał działać jak tylko do niego poszedłem (mam tę moc, mam tę moc!), udało mi się wyświetlić slajdy i opowiedzieć o zarządzaniu priorytetami projektów dokumentacyjnych. Podczas mojej blisko rocznej przygody w firmie Hitachi Energy miałem okazję przećwiczyć dostosowywanie priorytetów w dynamicznym środowisku, w którym tech writer ma więcej niż jeden projekt pod swoją pieczą. Cieszę się, że mogłem podzielić się obserwacjami, które mogą być pomocne również w kontekstach pozadokumentacyjnych, a także zaprezentowałem parę czerstwych memów własnego autorstwa.
+
+![](images/lee.png)
+
+Uczestnicy i uczestniczki zdarzenia okazywali zainteresowanie, zadali nawet pytania, a po zapaleniu świateł wszyscy byli obudzeni ;-) Dla mnie to definicja sukcesu.
+
+### Wejście Smoków
+
+![](images/1694696568105.jpg)
+
+Wisienką na torcie była rozmowa z sensei Mają Wójcik i sensei Jakubem Piaseckim, którzy opowiadali:
+* o dążeniu do mistrzostwa w dziedzinie karate Oyama
+* o pozytywnych wartościach sportowych jakie niesie ze sobą karate
+* o tym jak znajdować w sobie wewnętrzną siłę, by systematycznie rozwijać swoje umiejętności
+* o tym, jak przekonać siebie samego do pójścia na pierwszy trening
```

Comment:
```suggestion
Wisienką na torcie była rozmowa z sensei Mają Wójcik i sensei Jakubem Piaseckim, którzy opowiadali dążeniu do mistrzostwa w dziedzinie karate Oyama oraz pozytywnych wartościach sportowych jakie niesie ze sobą ten sport.

Dowiedzieliśmy się również o tym jak:
* Znajdować w sobie wewnętrzną siłę, by systematycznie rozwijać swoje umiejętności
* Przekonać siebie samego do pójścia na pierwszy trening
```

Again, nie wiem czy mamy jakiś style guide, ale MS mówi o zaczynaniu bulleted lists z dużej litery.

Reply:
nie mamy style guide ale jak sie chce to mozna zaszczepiac rozne dobre praktyki. dzieki temu jak mi zwrocisz na cos uwage moge sie zastanowic czy moje podejscie domyslne aby jest najlepsze :)